src/requirements.txt

```diff
@@ -1,6 +1,5 @@
 wheel==0.38.1
 docker==6.0.1
-certifi>=2023.7.22
```

Comment:
Why are we removing this?

Reply:
This was an attempt to fix the certifi issue, certifi installation is not a logical placement since requirement.txt is for environment set up: https://github.com/amazon-braket/amazon-braket-containers/commit/0108d8c142836e2aa65967326cd9fa4f54b8d40e